README.md

```diff
@@ -100,6 +100,57 @@ You are free to implement the genome classes however you want, and using whateer
 
 When you have implemented the two (or more) classes, describe the complexity of each operation as a function of the genome size (at the time of the operation), and the size of the TE involved (and when copying, the offset you are copying). Put the description here:
 
+
+
 **FIXME: OPERATION COMPLEXITY**
 
+length of the genome = n, 
+length of the transcriptional element you are workign with = t, 
+offset size = f, 
+number of active TEs= a 
+**Python List**
+
+***__init__***
+__init__'s complexity is O(n) because it has to run through all of self.genome which creates a list that is as long as the length of the genome, and the rest of the elements run in constant time 
+
+***insert_te***
+insert_te's complexity is O(n*t) because the most computationally demanding portion of this operation is in line 175 when the transcriptional element that is size t gets inserted into the genome that is the size n, and in doing so it has to push the rest of the genome down. In the worst case scenario, the TE would get inserted into the first index of the genome so the operation would have to run through the size of the transcriptional element and then the whole genome size to re-index every nucleotide, giving it a complexity of O(n*t)
+
+***copy_te***
+copy_te's complexity is also O(n*t) because the most computationally challenging portion of the operation is when it calls insert_te, which I think (as i said above) runs in O(n*t). the keys() method called on self.te has a complexity of O(a), which most likely  is going to be much smaller than the size of the genome and at worst case scenario would be the size of the genome so is not the portion of the operation that will be the most taxing. 
```

Comment:
Because `insert_te` actually costs $O(n+t)$, the complexity of this operation is also $O(n+t)$.